### PR TITLE
Add PyTest  suite for testing perl container in OpenShift 4

### PR DESCRIPTION
**Pull request by phracek**
This pull request supports testing perl container in OpenShift 4 by PyTest.

See a similar pull request here:
https://github.com/sclorg/s2i-python-container/pull/691

The pull request is separated into several commit in order to better review.
1) Adding test suite into `test` directory
2) Remove deprecated OpenShift 3 tests
3) Fix imagestream custom_tag definition
4) Distribution pytest into `<version>/test` symlinks.

### CODE REVIEW

**Comment by phracek** [comment]
[test-openshift-pytest]

File: 5.32/Dockerfile.rhel8 [hunk 1]
```diff
@@ -0,0 +1,66 @@
+FROM ubi8/s2i-base:1
```
Comment (michal-josef-spacek) [inline 5.32/Dockerfile.rhel8:1]: Duplicate with https://github.com/sclorg/s2i-perl-container/pull/308

Comment (phracek) [reply 5.32/Dockerfile.rhel8:1]: I know, I am waiting for merge #308

**Comment by phracek** [comment]
[test-openshift-pytest]

**Comment by phracek** [comment]
[test-all]

**Review by phracek — COMMENTED** [review]
The tests passed. Nothing is blocking. The pull request adds more tests with OpenShift 4 environment.

**Comment by phracek** [comment]
The pull request itself does not break s2i-perl-container functionality. Only adds new set of tests.!!!!

**Pull request merged**